src/controlnet_aux/zoe/zoedepth/models/base_models/midas_repo/hubconf.py

```diff
@@ -0,0 +1,435 @@
+dependencies = ["torch"]
+
+import torch
+
+from midas.dpt_depth import DPTDepthModel
```

Comment:
```suggestion
from .midas.dpt_depth import DPTDepthModel
```

shoudln't we do local imports instead of absolute?